test/integration/targets/mongodb_replicaset/aliases

```diff
@@ -0,0 +1,4 @@
+destructive
+shippable/posix/group1
```

Comment:
Add the `needs/root` alias for both tests.

Reply:
Done.